alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -532,12 +532,13 @@ class MonitorRunner(
                     if (!destination.isAllowed(allowList)) {
                         throw IllegalStateException("Monitor contains a Destination type that is not allowed: ${destination.type}")
                     }
-
+                    val denyHostRanges = settings.getAsList("opendistro.destination.denylist", listOf())
```

Comment:
It doesn't look like we're registering these settings anywhere. We can register it under `DestinationSettings` so the setting can be set/used. Let's also add an integration test with this being used.

Also, we might want to consider something a little different than `"opendistro.destination.denylist"`, since we have an `"opendistro.alerting.destination.allow_list"` setting already.

Reply:
+1 for putting this in `DestinationSettings`
Yea we should rename `opendistro.destination.denylist`. Maybe we can use `opendistro.alerting.destination.deny_ip_list`. We need to add IP to the setting to make it more clear what it is for since the allow_list also pertains to something a bit different.